src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -57,11 +57,15 @@
       <Architectures>x64</Architectures>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="ios">
+      <Parent>unix</Parent>
+      <Architectures>arm</Architectures>
+      <Versions>10</Versions>
```

Comment:
```suggestion
      <Versions>8;9;10</Versions>
```

Reply:
We actually go back to iOS 7 for ARM/ARM64.

Reply:
What about x64?  Marek's suggestion below indicates that's also supported back to 7.

Reply:
Let's start at 8 everywhere